examples/next/pages/ui/components/geo/use-map-hook/index.page.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, MapView } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+import { useMap } from 'react-map-gl';
+
+import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+
+Amplify.configure(awsExports);
+
+function FlyToButton() {
+  const { current: map } = useMap();
+
+  const flyToMordor = () => {
+    map.flyTo({ center: [172.78, -42.28], zoom: 5 });
```

Comment:
if we always use the same `flyTo` number, maybe we should consider only using one constant?

Reply:
The two examples are meant to be independent of each other so shouldn't share a constant value.